aten/src/ATen/cpu/vec256/missing_vld1_neon.h

```diff
@@ -259,194 +259,194 @@ __extension__ extern __inline void
 __attribute__ ((__always_inline__, __gnu_inline__, __artificial__))
 vst1_s64_x2 (int64_t * __a, int64x1x2_t val)
 {
-  asm ("st1 {%S0.1d - %T0.1d}, [%1]" :: "w" (val), "r"(__a) :);
+  asm ("st1 {%S1.1d - %T1.1d}, %0" : "=Q" (*__a) : "w" (val) : "memory");
```

Comment:
curious why do we need to switch registers for `st` but not `ld`?

Reply:
No particular reason(as you can see order of operands was changed, but the same is true about order of parameters), important parts here is "=Q" and "memory" modifiers